docs/en/backup-restore.rst

```diff
@@ -2,56 +2,169 @@
+
+.. note::
+  
+  The work factor for PBKDF2 is implemented through an iteration count, which should set differently based on the internal hashing algorithm used. The recommended value for ``SHA-256`` hashing algorithim is 600,000 iterations as stated in the `OWASP Password Storage Cheatsheet <https://cheatsheetseries.owasp.org/cheatsheets/Password_Storage_Cheat_Sheet.html#pbkdf2>`_.
+
+
+**Step 4**: The Wallet Instance performs the operations below to create the backup JWT entry for the backup file. 
+ 
+- For each of the HW bound key credentials, add the ``iss``, ``credential_configuration_id`` as an entry in the backup JWT. 
+- Sign the backup JWT using the private key that its public key is attested within the Wallet Attestation. The related public key that is attested by the Wallet Provider is provided within the Wallet Attestation (``cnf`` claim). The Wallet Instance MUST verify the validity of the Wallet Attestation before signing the backup JWT.
+- Add the signed backup JWT as an entry to the backup file. 
+- Encrypt the backup file using the provided key phrases. 
+
+.. note::
+
+  The Backup JWT MAY contain transaction history for each credential entry within the ``credentials_backup`` claim.
+
+  
+
+**Step 5**: The User will be prompted to select the storage for securely storing the backup file based on his preference. This can range from native storage to external storage (e.g., cloud storage, usb, etc.). 
+
+**Step 6**: Considering the native storage as the preferred choice, the file will be stored on the User device.
+
+A non-normative example of the backup JWT is as the following:
+ 
+ .. code-block::
+  
+  { 
+    "alg": "ES256",   
+    "kid": "pDV1n6WNT4qFx2JbTv7S66WE55BkmHNyuEKO4JctiPw",
+    "typ": "wallet_instance_backup+jwt",    
+  }
+  .
+  {
+    "timestamp":"2024-12-13T16:35:06+01:00",
+    "wallet_provider_id":"https://wallet-provider.example.org/",
+    "wallet_instance_version":"v1.0",
+    "wallet_attestation":"eyJhbGciOiJFUzI1NiIsImVVfQz.eyJpc3MiOiAiaH...LCAibmJ",
+    "credentials_backup":[
+     {  
+        "iss": "https://issuer.example.org/v1.0/mdl",
+        "credential_configuration_id":"org.iso.18013-5.1.mDL"
+     },
+     {  
+        "iss": "https://eaa-provider.example.org/",
+        "credential_configuration_id":"EuropeanDisabilityCard"
+     },
+   ]
+  }
+
+The JOSE header of the backup JWT MUST contain the following REQUIRED parameters:
```

Comment:
```suggestion
The JOSE header of the backup JWT MUST contain the following REQUIRED parameters:
```

why do we use JWT and not CWT? We would reduce the payload size

Reply:
It becomes more complicated to manage and requires ad-hoc implementations that we do not have for now. It is much simpler to use a JWT because the difference in terms of size is really negligible. We are talking about a few Kb